src/main/java/util/DataConverterImpl.java

```diff
@@ -0,0 +1,26 @@
+package util;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> lines) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : lines) {
+            String[] parts = line.split(",");
+            FruitTransaction.Operation operation = FruitTransaction.Operation.fromCode(parts[0]);
+            String fruit = parts[1];
+            int quantity = Integer.parseInt(parts[2]);
+
+            FruitTransaction transaction = new FruitTransaction();
+            transaction.setOperation(operation);
+            transaction.setFruit(fruit);
+            transaction.setQuantity(quantity);
+
+            transactions.add(transaction);
+        }
+        return transactions;
+    }
+}
```

Comment:
The class `DataConverterImpl` lacks proper encapsulation. The class and its method should be declared with the appropriate access modifiers to hide its implementation details. Since this class is an implementation of a service, it should be `public`, but if it's not intended to be used outside of its package, you could consider package-private visibility.